src/readonlyKeywordRule.ts

```diff
@@ -49,10 +49,13 @@ function checkPropertySignatureAndIndexSignature(
     if (Ignore.shouldIgnorePrefix(node, ctx.options, ctx.sourceFile)) {
       return [];
     }
+
+    const start = utils.isIndexSignatureDeclaration(node)
```

Comment:
`readonly` seems to always be the last keyword for members, hence inserting just before member name. Index signatures don't have names.